STYLEGUIDE.md

```diff
@@ -0,0 +1,28 @@
+As with code, technical writing benefits from consistent rules. Please follow this guide when adding or editing CircleCI documentation.
+
+# Structure
+
+- Don't use h1 headers. The title you choose in a doc's frontmatter will automatically be converted into an h1.
+- Add a table of contents to articles with more than 10 h2s.
```

Comment:
Cool, this is already in CONTRIBUTING.md, so this will be solved when I move it there.